src/components/m-header-mobile/index.scss

```diff
@@ -87,24 +87,32 @@
   // https://github.com/axa-ch/patterns-library/issues/263
   background: $color-prim-white;
 
-  > .m-button {
-    display: block;
-    margin: $header-mobile__item-margin-top $header-mobile__item-padding-right 0 $header-mobile__item-padding-left;
-
-    text-align: center;
+  .m-header-top-content-bar__box {
+    padding-left: $header-mobile__item-padding-left;
+    padding-right: $header-mobile__item-padding-right;
+  }
 
+  axa-button {
```

Comment:
Sure you want to rely on tag names?

Reply:
not yet :) but only button is accepted so either class on axa-button or axa-button itself. What is your suggestion?

Reply:
Yeah that's again an issue with the additional wrapping DOM node.
I would rather fix #674 

But for now, if you can make it a class, perfect.
If not add a todo please

Reply:
I can make a class is just less user friendly and more likle that an error happens as the user of it have to add that class as well.

Reply:
Well originally defined how we selected and style DOM nodes:
https://github.com/axa-ch/patterns-library/blob/develop/CONTRIBUTING.md#css

But I see the complexity of this special case, caused by not using built-ins #674 

So please at least add the todo

Reply:
Oky doky